src/modules/visits/services/visits.service.ts

```diff
@@ -249,7 +259,7 @@ export class VisitsService {
 			limit,
 			orderBy: set(
 				{},
-				ORDER_PROP_TO_DB_PROP[orderProp || 'startAt'],
+				ORDER_PROP_TO_DB_PROP[orderProp] || ORDER_PROP_TO_DB_PROP['startAt'],
```

Comment:
How is this better than what it was before? It's more code and does the exact same thing without making it much clearer.

Reply:
`startAt` !== `start_date`

`startAt` is not a valid value for `orderBy`